### PR TITLE
[ja] Change paths of debug pages (case 3)

### PR DESCRIPTION
**Pull request by shogohida**
Fixes https://github.com/kubernetes/website/issues/38797

This PR changes the paths of debug pages. The example shown on the issue is below.

- Change path and add "/ja"
  - e.g.: /docs/tasks/debug-application-cluster/resource-usage-monitoring/ -> /ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/

### CODE REVIEW

File: content/ja/docs/concepts/cluster-administration/manage-deployment.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 reviewers:
-- 
+-
```
Comment (shogohida) [inline content/ja/docs/concepts/cluster-administration/manage-deployment.md:3]: Blank space was deleted by Visual Studio Code

**Comment by shogohida** [comment]
@t-inu 
I created the last PR! I have a question. There is a wrong path (`/docs/tasks/debug-application-cluster/troubleshooting/`) in [this file](https://github.com/kubernetes/website/blob/main/content/ja/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md) but there is not a file called `troubleshooting`. What should I do?

**Comment by t-inu** [comment]
@shogohida 
The Windows page has been changed by PR https://github.com/kubernetes/website/pull/33582, and deleted `intro-windows-in-kubernetes.md`  in English.
When this page existed, it was described as follows.
```
Your main source of help for troubleshooting your Kubernetes cluster should start with the [Troubleshooting](/docs/tasks/debug/) page.
```
Currently, I think the following update would be good for Japanese page.
```
Kubernetesクラスターのトラブルシューティングの主なヘルプソースは、[トラブルシューティング](/ja/docs/tasks/debug/)ページから始める必要があります。
```

**Comment by shogohida** [comment]
@t-inu 
Thanks for your comment! I changed the code so please check it

File: content/ja/docs/tasks/debug/debug-cluster/local-debugging.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ content_type: task
 
 <!-- overview -->
 
-Kubernetesアプリケーションは通常、複数の独立したサービスから構成され、それぞれが独自のコンテナで動作しています。これらのサービスをリモートのKubernetesクラスター上で開発・デバッグするには、[get a shell on a running container](/docs/task/debug-application-cluster/get-shell-running-container/)してリモートシェル内でツールを実行しなければならず面倒な場合があります。
+Kubernetesアプリケーションは通常、複数の独立したサービスから構成され、それぞれが独自のコンテナで動作しています。これらのサービスをリモートのKubernetesクラスター上で開発・デバッグするには、[get a shell on a running container](/ja/docs/task/debug/debug-application/get-shell-running-container/)してリモートシェル内でツールを実行しなければならず面倒な場合があります。
```
Comment (t-inu) [inline content/ja/docs/tasks/debug/debug-cluster/local-debugging.md:8]:
```suggestion
Kubernetesアプリケーションは通常、複数の独立したサービスから構成され、それぞれが独自のコンテナで動作しています。これらのサービスをリモートのKubernetesクラスター上で開発・デバッグするには、[実行中のコンテナへのシェルを取得](/ja/docs/tasks/debug/debug-application/get-shell-running-container/)してリモートシェル内でツールを実行しなければならず面倒な場合があります。
```

Comment (shogohida) [reply content/ja/docs/tasks/debug/debug-cluster/local-debugging.md]: Thanks for your review! I fixed the content :bow:

https://github.com/kubernetes/website/pull/39528/commits/c6809f4d79f84f67daaf2e1e8dc5cab02b01ea15

**Review by t-inu — APPROVED** [review]
Thank you!
/lgtm

File: content/ja/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -387,7 +387,7 @@ GAになってからさらなる変更を加えることは現実的ではない
 - `CustomResourceWebhookConversion`: [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのWebhookベースの変換を有効にします。
 - `DevicePlugins`: [device-plugins](/docs/concepts/cluster-administration/device-plugins/)によるノードでのリソースプロビジョニングを有効にします。
 - `DryRun`: サーバーサイドでの[dry run](/docs/reference/using-api/api-concepts/#dry-run)リクエストを有効にします。
-- `DynamicAuditing`: [動的監査](/docs/tasks/debug-application-cluster/audit/#dynamic-backend)を有効にします。
+- `DynamicAuditing`: [動的監査](/docs/tasks/debug/debug-cluster/audit/#dynamic-backend)を有効にします。
```
Comment (atoato88) [inline content/ja/docs/reference/command-line-tools-reference/feature-gates.md:390]: Just information.
It's not the target of this PR.
This PR looks ok for me.

`DynamicAuditing` is deprecated on 1.19 and [removed](https://github.com/kubernetes/website/commit/c52731f462f3412fabdbe723151fd1f95bc18127) as of now.

Comment (shogohida) [reply content/ja/docs/reference/command-line-tools-reference/feature-gates.md:390]: Oh, I didn't know that. Thanks for your information!

Comment (atoato88) [reply content/ja/docs/reference/command-line-tools-reference/feature-gates.md:390]: We have a few:sweat_smile: pages needed to be updated.

Comment (t-inu) [reply content/ja/docs/reference/command-line-tools-reference/feature-gates.md:390]: `DynamicAuditing` doesn't exist in the latest English [Feature Gates page](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
In Japanese page, the description is still there, just deleted from the list.
So, whether fix the link or not, this line will be deleted eventually in Japanese, too.
I think delete this line by this PR is also good.

Comment (shogohida) [reply content/ja/docs/reference/command-line-tools-reference/feature-gates.md]: I thought it was good to delete this line so I deleted it. Please review again 🙇 

https://github.com/kubernetes/website/pull/39528/commits/516442986aaa57cfaf4c26c4c7f579772438dd0f

**Comment by t-inu** [comment]
@shogohida 
Deploy failed has occurred.
The build log is similar to the following comment.
https://github.com/kubernetes/website/pull/39479#issuecomment-1448520224

And this comment may help you.
> Try rebasing this PR against *main*.

https://github.com/kubernetes/website/pull/39479#issuecomment-1448520599

**Comment by t-inu** [comment]
The cause of the error has changed.
"Build exceeded maximum allowed runtime"
Here is a same case.
https://github.com/kubernetes/website/pull/39479#issuecomment-1446408971

**Comment by shogohida** [comment]
@t-inu @atoato88 
Thanks for your comments! I made an empty commit and pushed it and checks passed!

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**